clean-abap/CleanABAP.md

```diff
@@ -4163,6 +4164,38 @@ class hiring_test defintion
 endclass.
 ```
 
+#### Put help methods in help classes
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Testing](#testing) > [Test Classes](#test-classes) > [This section](#put-help-methods-in-help-classes)
+
+Put help methods used by several test classes in an abstract help class.
```

Comment:
I would not limit help methods as abstract parent classes. This may lead to the habit to have unrelated code in the helper class, instead of having dedicated helper classes for them with delegation.

Reply:
Thanks, that is a valid point. I rewrote it a bit. Please let me know your thoughts.